pom.xml

```diff
@@ -23,47 +23,166 @@
         <module>bstats-sponge-lite</module>
     </modules>
 
+    <scm>
+        <connection>scm:git:https://github.com/Bastian/bStats-Metrics.git</connection>
+        <developerConnection>scm:git:git@github.com:Bastian/bStats-Metrics.git</developerConnection>
+        <url>https://github.com/Bastian/bStats-Metrics</url>
+    </scm>
+
+    <ciManagement>
```

Comment:
This is not even officially used by Bastian, why are you adding it? Same for the distribution management down below - bStats-Metrics has already moved to jitpack.io

Reply:
As i talked privately with bastian, jitpack is causing some issues in some scenarios.

Reply:
Could you describe those issues in more detail? I haven't had any for the moment.